README.md

```diff
@@ -128,7 +119,7 @@ This project will consist of five main sprints and two smaller sprints. The proj
     * Students are able to memorize their passwords for different websites.
 * Assignment
   * Students can log in to a new account after 1 day, 1 week, and 1 month of creating the password.
```

Comment:
Might be difficult to measure effectively, not that it's any less valuable or that we can't change it in the future